src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantSourceService.java

```diff
@@ -20,7 +20,7 @@
         return convert(repository.findAll());
     }
 
-    private List<VariantSource> convert(List<VariantSourceMongo> variantSources) {
```

Comment:
Can we make the findAll method to return List<IVariantSource> instead of List<VariantSource>? We wouldn' t need the convert method if that is possible.